templates/federated_reporting/import_file.sh

```diff
@@ -22,9 +26,21 @@ hostkey=$(basename "$file" | cut -d. -f1)
 
 table_whitelist=$(printf "'%s'," $CFE_FR_TABLES | sed -e 's/,$//')
 
+psql_debug_option="--quiet"
+
+if $CFE_FR_DEBUG; then
```

Comment:
Would be better and safer to just compare strings here.

Reply:
in terms of command injection or something? makes sense in any case.

Reply:
Yeah, there's no reason to execute it as a command here, I think.